### ISSUE TITLE
Feature: Game  Notation

### ISSUE THREAD

**Issue by Segelzwerg** [Whist-Team/Whist-Core#243]
There should be a game notation for each hand like there is in [PGN in chess](https://en.wikipedia.org/wiki/Portable_Game_Notation).

I propose a similar structure based on TOML. It maybe even applicable for other games like Skat, Doppelkopf etc.

Header shall contain list of players in order of seating with current rating and team associated. The type of game is required with some Whist specifics like hand number of rubber. Also trump is necessary.

The data is a list of tricks.
```toml
<Trick number>: <Player number>: <Suit><Rank>
```

**Comment by iTitus** [comment]
Interesting.
I'd vouch for the structure of a known format. (TOML, JSON, YML, XML, Lua, EDN, ...), so we don't have to write and debug a custom parser.

**Comment by Segelzwerg** [comment]
Agree, would prefer for it's human readability and it will be part of std lib from 3.11 on. Do you prefer something else @iTitus ?

**Comment by iTitus** [comment]
Depends on our goals.
Do we want want human readability/writability?
This plays into the desired verbosity and leniency of the parser and thus the format.
